tests/test_data_fillers.py

```diff
@@ -0,0 +1,92 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('..'))
+
+from unittest import TestCase
+from modules.DMZ.data_kit import data_io
+from modules.DMZ.data_kit import data_filler
+import setup
+import numpy as np
+import pandas as pd
+
+from modules.toolbox.data_package import DataPackage
+from modules.toolbox.validation_package import ValidationPackage
+
+class TestDataFilling(TestCase):
+
+    def test_drop_missing_data_columns(self):
+
+        #Arrange
+        data, start, target = self.setup_data()
+
+        #Act
+        end = (data_filler.drop_missing_data_columns(data, len(data))).shape
+
+        #Assert
+        self.assertLess(end[1], start[1], msg="Failed to beat baseline")
```

Comment:
Tests are looking good, but I'd suggest changing these failure errors to be more descriptive of the real problem if it fails.